java/openjdk17/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 
 name                openjdk17
 # https://github.com/openjdk/jdk17u/tags
-version             17.0.7
-set build 7
+version             17.0.8
+set build 2
```

Comment:
jdk-17.0.8+3 was released the last week ;)

Reply:
Rebased, thanks!